src/lib/isIdentityCard.js

```diff
@@ -209,20 +203,19 @@ const validators = {
     const check15IdCardNo = (idCardNo) => {
       let check = /^[1-9]\d{7}((0[1-9])|(1[0-2]))((0[1-9])|([1-2][0-9])|(3[0-1]))\d{3}$/.test(idCardNo);
       if (!check) return false;
-      let addressCode = idCardNo.substring(0, 6);
+      let addressCode = idCardNo.substring(0, 2);
```

Comment:
Add a note too about this change.

Reply:
If you check the old implementation of ```checkAddressCode``` you can see that it accepts a 6 character string (addressCode), do a regexp check on it's length and verify if the first 2 characters are a valid provinceCode.
Since length check is unnecessary and already done in line 204, i am directly passing the provinceCode to ```checkAddressCode``` and it a 2 characters numerical string